taxi_service/settings.py

```diff
@@ -140,3 +141,5 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+CRISPY_TEMPLATE_PACK = "bootstrap4"
```

Comment:
The `CRISPY_TEMPLATE_PACK` is set to `bootstrap4`. Ensure that the `crispy_forms` package is installed and configured correctly to use Bootstrap 4 for form rendering.